### PR TITLE
Issue273 testcase multizone office simple air

### PR DESCRIPTION
**Pull request by dhblum**
This is for #273.  In particular, it implements the test case.

@JavierArroyoBastida The detailed test case review process with @Mathadon has completed and I have prepared the test case for merging.  Do you mind briefly reviewing this though to make sure everything is in order?

### CODE REVIEW

**Review by JavierArroyoBastida — CHANGES_REQUESTED** [review]
Thanks a lot, @dhblum and @Mathadon for the continued efforts! 
It is looking good to me. I would just suggest to add this emulator to the [compare_references.py file](https://github.com/ibpsa/project1-boptest/blob/09e60d39d853fbf10df6673c8b023bbb7a6ea4e1/testing/compare_references.py#L126) with a few critical/representative variables of this model. I believe this is good practice for future revisions to check on potential reference changes.

**Comment by dhblum** [comment]
@JavierArroyoBastida Thanks for the review!  ``compare_references.py`` updated in https://github.com/ibpsa/project1-boptest/pull/412/commits/aa5a1e3fd188c0a8580bae9e65373159d166a836.

File: data/find_days.py [hunk 1]
```diff
@@ -39,6 +39,12 @@ def find_days(heat, cool, data='simulate', img_name='boptest_bestest_air',
         performed.
     plot: boolean
         Set to True to show an overview of the days found
+    peak_cool_restriction_hour: integer, optional
```
[the remark this answers is not in the document]
Comment (dhblum) [reply data/find_days.py:42]: I suggest waiting until needed.

[the remark this answers is not in the document]
Comment (JavierArroyoBastida) [reply data/find_days.py:42]: Ok makes sense!

**Comment by JavierArroyoBastida** [comment]
I see. Looking great, thanks! I'd suggest merging once tests pass.

**Review by JavierArroyoBastida — APPROVED** [review]

**Pull request merged**